index.js

```diff
@@ -0,0 +1,177 @@
+const inquirer = require("inquirer");
+const db = require("./database/connection.js");
+require("console.table");
+
+async function start() {
+  mainQuestions();
+}
+
+async function mainQuestions() {
+  const res = await inquirer.prompt([
+    {
+      type: "list",
+      name: "Choices",
+      message: "Please choose one of the options or add:",
+      choices: [
+        "View all departments",
+        "View all roles",
+        "View all employees",
+        "Add a department",
+        "Add a role",
+        "Add an employee",
+        "Update an employee role",
+        "Exit",
+      ],
+    },
+  ]);
+  let choice = res.Choices;
+  switch (choice) {
+    case "View all departments":
+      viewDepartments();
+      break;
+    case "View all roles":
+      viewRoles();
+      break;
+    case "View all employees":
+      viewEmployees();
+      break;
+    case "Add a department":
+      addDepartment();
+      break;
+    case "Add a role":
+      addRole();
+      break;
+    case "Add a employee":
```

Comment:
The reason the app freezes when a user tries to add an employee or update one is becuase there isn't an appropriate switch case for updating and for adding on there's an 'n' missing here. It should say: 
```
case "Add an employee"
```